Cyborg themed bags

SirYoshimoto opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Hello all! First PR!
Adds in 10 Cyborg themed backpacks. They function as regular backpacks.

Mining cyborg bag
Janitor cyborg bag
Peace cyborg bag
Service cyborg bag
Security cyborg bag
Medical cyborg bag
Engineer cyborg bag
Syndicate Assault cyborg bag
Syndicate Medical cyborg bag
Syndicate Engineer cyborg bag

All bags except for syndicate cyborg bags will be available in department clothes vendors for 300cr and the beecoin shop for 2000 becoins!
Syndicate bags will be available in the donksoft vendor for 350cr and the donor beecoin shop for 3500 beecoins.



## Why It's Good For The Game

A way to immerse oneself into the games universe (similar to the effect of plushies), potentially increasing RP via deeper immersion.
In-universe representation of these machines in a form that NT would use as a way to commercialize, popularize and produce support for the stations hardworking cyborgs.
And to increase the variety of backpacks currently present in-game.


## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>
<summary>Screenshots&Videos</summary>

<img width="672" height="112" alt="Screenshot 2025-07-20 202929" src="https://github.com/user-attachments/assets/57200e42-035f-4d1f-9b63-a145612e2064" />

<img width="727" height="601" alt="image" src="https://github.com/user-attachments/assets/9ce2cef7-9d6e-44ac-9179-aa56b8746ecc" />

https://github.com/user-attachments/assets/180eedfe-a81e-45e0-a2fe-0d68e900ccbe




</details>

## Changelog
:cl: Doktorwueue (wueue) and SirYoshimoto
add: Adds crew wearable backpacks in the style of all cyborgs to Cargo, Security, Engineering, Medical, Bar, Chef, Botany, Donksoft and Robotics clothes vendors for 300cr (Syndicate bags are 350cr). Also available in the Beecoin shop for 2000 beecoins (syndicate bags are 3500 beecoins for donors only).
imageadd: Adds mob, inhand and icon sprites of 10 Cyborg style backpacks.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

SirYoshimoto committed:
Cyborg Themed Backpacks

Adds 10 Cyborg themed backpacks.
*Mining cyborg bag
*Janitor cyborg bag
*Peace cyborg bag
*Service cyborg bag
*Security cyborg bag
*Medical cyborg bag
*Engineer cyborg bag
*Syndicate Assault cyborg bag
*Syndicate Medical cyborg bag
*Syndicate Engineer cyborg bag

SirYoshimoto committed:
Update wardrobes.dm

SirYoshimoto committed:
Update toys.dm

Nakalon commented on code/game/objects/items/storage/backpack.dm:
```diff
@@ -241,6 +241,57 @@
+	name = "Engineering Cyborg bag"
+	desc = "Store the tools to repair that breach in this cute borgiebag!"
+	icon_state = "engiborgpack"
+	resistance_flags = FIRE_PROOF
```
Funny you added fireproof to this. I like it as a little easter egg.

SirYoshimoto replied:
I thought it would make sense since most of the other engi related stuff is fire proofed too!

Nakalon commented on code/modules/client/loadout/loadout_accessories.dm:
```diff
@@ -249,3 +249,37 @@
+
+/datum/gear/accessory/backpack
+	subtype_path = /datum/gear/accessory/cloak
+	cost = 2000
```
Can you give us a photo of these on the bee shop so we know all is okey?

SirYoshimoto replied:
Oh I forgot to add that! Fixed!

Nakalon replied:
> Oh I forgot to add that! Fixed!

Mark my comments as resolved

Nakalon commented on code/modules/vending/toys.dm:
```diff
@@ -16,7 +16,10 @@
 		/obj/item/clothing/suit/syndicatefake = 5,
-		/obj/item/clothing/head/syndicatefake = 5)
+		/obj/item/clothing/head/syndicatefake = 5
+		/obj/item/storage/backpack/syndiassaultborg = 2,
```
This:

/obj/item/clothing/head/syndicatefake = 5
		/obj/item/storage/backpack/syndiassaultborg = 2,
		
		Should be 

/obj/item/clothing/head/syndicatefake = 5,
		/obj/item/storage/backpack/syndiassaultborg = 2
		
		
		you put the comma in the wrong place! Plus you'll have to close it with ) probably

SirYoshimoto committed:
Update toys.dm

SirYoshimoto committed:
check tester

git dem check test WORRRKIIIINN OOUURGGHH WOOOGA WOOOGA
IGNORE THIS IGNORE THIS

DrDuckedGoose commented on code/game/objects/items/storage/backpack.dm:
```diff
@@ -241,6 +241,57 @@
 	icon_state = "ert_engineering"
 
+/obj/item/storage/backpack/engineerborg
+	name = "Engineering Cyborg bag"
```
I could be mistaken but-
You'll probably want this, and the other's to be ``"\improper Cyborg Bag"``
Otherwise, I think, it will show up as ``That's Cyborg Bag!`` instead of ``That's a Cyborg Bag!``

SirYoshimoto replied:
Oh you mean it showing up as a bag for cyborgs instead of cyborg themed bags?

Rukofamicom replied:
No the grammar will be wrong in examine messages.

SirYoshimoto replied:
Ah I see, I shall correct it!

DrDuckedGoose commented on code/game/objects/items/storage/backpack.dm:
```diff
@@ -241,6 +241,57 @@
+
+/obj/item/storage/backpack/syndiengborg
+	name = "Suspicious Engineering Cyborg bag"
+	desc = "Store the tools that will lead to the demise of Nanotrasen in an adorable bag! A smell of omnizine infuzed oil emanates from it."
```
Seems the others use ``this`` instead of an. 
I can see some of the others mix and match it, so it's your call.
```suggestion
	desc = "Store the tools that will lead to the demise of Nanotrasen in this adorable bag! A smell of omnizine infuzed oil emanates from it."
```

SirYoshimoto replied:
I'll probably just make it so it's just the Syndie borg bags that use `an` just to add to the rebel nature of them.

SirYoshimoto committed:
Grammer correction+fireproofed syndiengiborgbag

Corrects "improper" things related to the cyborg bags
Also adds fireproof to the syndicate engineer cyborg bag

DrDuckedGoose reviewed (approved):
LGTM

PowerfulBacon reviewed (changes requested):
Borg bags in standard vendors should be under the premium tab and have a premium vendor price

SirYoshimoto committed:
Moved bags to Premium

Moved standard bags to the premium section of vendors.

SirYoshimoto committed:
Price + Shop modification

Added standard price to all normal bags 300cr.
Syndie bags are 350cr.

Added syndie bags to beecoin shop for 5000 beecoins, Free for donors.

SirYoshimoto committed:
removed syndi bags from normal beeshop

Removed syndi bags from the normal beecoin shop, now available for donors only

SirYoshimoto committed:
File rearrangement

Cleaned up syndi bag location in loadout_donator.dm
